### PR TITLE
added failed alchemy call to queue

### PR DESCRIPTION
**Pull request by kailash-bitpack**

### CODE REVIEW

File: apps/server/src/consumer/queues/bullBoard.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { createBullBoard } from 'bull-board';
+import { BullMQAdapter } from 'bull-board/bullMQAdapter';
+
+import { registerQueues } from './queue';
+
+const { router } = createBullBoard(registerQueues().map(qd => new BullMQAdapter(qd.q)))
+export const bullBoard = (app) => {
+    app.use('/admin/queues', router);
```
Comment (r4881t) [inline apps/server/src/consumer/queues/bullBoard.ts:8]: This should be behind some kind of authentication

File: apps/server/src/consumer/queues/config.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    host: process.env.REDIS_HOST,
+    port: parseInt(process.env.REDIS_PORT as string),
+    username: process.env.REDIS_USERNAME as string,
+    password: process.env.REDIS_PASSWORD as string
+  }
+};
+
+const queues = new Map();
+
+export const createQueueMQ = (name: string) => {
```
Comment (r4881t) [inline apps/server/src/consumer/queues/config.ts:17]: Rename this to `createOrGetQueue` so that it correctly signifies the behavior.

**Review by r4881t — COMMENTED** [review]
Won't you need to setup the Queue Processors to start at server start? Something like https://github.com/bitpack-me/EntityBit/blob/fce731cfe5135f0fdfbaa025e0423933b0126f4e/src/lib/queues/Worker.ts#LL109C1-L109C1

**Comment by kailash-bitpack** [comment]
> Won't you need to setup the Queue Processors to start at server start? Something like https://github.com/bitpack-me/EntityBit/blob/fce731cfe5135f0fdfbaa025e0423933b0126f4e/src/lib/queues/Worker.ts#LL109C1-L109C1

from bullMQ 2.0.0 QueueScheduler is depricated.

**Review by r4881t — APPROVED** [review]

**Review by r4881t — APPROVED** [review]

**Pull request merged**